src/compass/utils/lut.py

```diff
@@ -362,3 +367,55 @@ def resample_set(geo_tide, pts_src, pts_dest):
                    bounds_error=False, fill_value=0)
     rdr_tide = rgi_func(pts_dest)
     return rdr_tide
+
+
+def ionosphere_delay(sensing_time, wavelength,
```

Comment:
The input of the function should be generic. I assume here you need a date to be provided in a certain format not necessarily the burst sensing start.

Reply:
[`isce3`](https://github-fn.jpl.nasa.gov/isce-3/isce/blob/4694efe362713b3af1d374270ebfaebf469c966a/python/packages/isce3/atmosphere/tec_product.py#L15) uses UTC time.

Also, I think this function is a better fit in [iono.py](https://github.com/opera-adt/COMPASS/blob/main/src/compass/utils/iono.py).

Reply:
Thanks for the suggestion. I agree the placement of this function, and generic input for the time component of this function. For that I've moved the function into `iono.py`, and renamed `sensing_time` into `utc_time`
For the data type of `utc_time` I think I prefer to keep `datetime.datetime`. What do you think?

Reply:
Yeah, I like having the data type of `utc_time` to be `datatime.datetime` (let's clarify it in a comment or in the docstring), what do you think @LiangJYu ?

Reply:
`datetime.datetime` sounds good to me.